assets/app/view/game/part/track.rb

```diff
@@ -29,58 +29,36 @@ def render
           # Array<Array<Path>>
           @routes_paths = @routes.map { |route| route.paths_for(@tile.paths) }
 
-          if @tile.offboards.any?
-            @tile.paths.select(&:offboard).map do |path|
+          tracks = @tile.paths.select { |path| path.edges.size == 2 }
+          .map { |path| [path, index_for(path)] }
+          .sort_by { |_, index| index || -1 }
+          .map do |path, index|
+            h(TrackCurvilinearPath, region_use: @region_use, path: path,
+                                    color: value_for_index(index, :color),
+                                    width: value_for(path, :width),
+                                    dash: value_for(path, :dash),)
+          end
+
+          tracks << @tile.paths.map do |path|
+            if path.offboard
               h(TrackOffboard, offboard: path.offboard, path: path, region_use: @region_use,
                                color: value_for(path, :color), width: value_for(path, :width),
                                dash: value_for(path, :dash),)
-            end
-          elsif @tile.lawson?
-            @tile.paths.select { |path| path.edges.one? }.map do |path|
-              h(TrackLawsonPath, path: path, region_use: @region_use,
-                                 color: value_for(path, :color), width: value_for(path, :width),
-                                 dash: value_for(path, :dash),)
-            end
-          elsif @tile.towns.any?
-            render_track_for_curvilinear_town
-          elsif @tile.cities.any?
-            render_track_for_curvilinear_city
-          else
-            @tile.paths.select { |path| path.edges.size == 2 }
-            .map { |path| [path, index_for(path)] }
-            .sort_by { |_, index| index || -1 }
-            .map do |path, index|
-              h(TrackCurvilinearPath, region_use: @region_use, path: path, color: value_for_index(index, :color),
-                                      width: value_for(path, :width), dash: value_for(path, :dash),)
-            end
-          end
-        end
-
-        private
-
-        def render_track_for_curvilinear_city
-          @tile.cities.flat_map do |city|
-            exits = city.exits
-
-            city.paths.map do |path|
-              h(TrackCurvilinearHalfPath, exits: exits, path: path, region_use: @region_use,
+            elsif path.junction || path.city || (path.town && path.town.exits.size != 2)
```

Comment:
Why exclude towns with 2 exits here? Seems like it would be better for those to use `TrackNodePath` and get rid of `TrackCurvilinearHalfPath`. I think that would fix #264

Reply:
I may eventually do that, but it would require changing town rendering to make sure they land on top of the bezier curves.

Reply:
Is it OK if I hold off for a bit on this specific change?

Reply:
No problem!